autometa/common/coverage.py

```diff
@@ -212,11 +213,11 @@ def align_reads(
         # Now we need to determine which point to start the calculation...
         for fp, argname in zip([bed, bam, sam], ["bed", "bam", "sam"]):
             step = "full"
-            if os.path.exists(fp):
+            if os.path.exists(fp) and os.path.getsize(out):
                 step = f"{argname}_exists"
                 break
 
-        if (not fwd_reads or not rev_reads) and step == "full":
+        if ((not fwd_reads or not rev_reads) and not se_reads) and step == "full":
```

Comment:
Single end reads are not necessarily required.

Reply:
I added this because there could be a case when the user just provided the `se_reads` and not the `fwd_reads` and `rev_reads`. If I removed, ` and not se_reads` then it will throw an error even if the user-provided `se_reads`

Reply:
I have tried this with only single-end reads. Do you know if the calculation sequence will work with only single-end reads?